src/wp-includes/class-wp-widget.php

```diff
@@ -153,7 +153,7 @@ public function form( $instance ) {
 	 * @since 2.8.0
 	 *
 	 * @param string $id_base         Optional. Base ID for the widget, lowercase and unique. If left empty,
-	 *                                a portion of the widget's PHP class name will be used. Has to be unique.
+	 *                                a portion of the widget's PHP class name will be used. Has to be unique. Default partial class name.
```

Comment:
I'm not sure this should have `Optional`. It's a required parameter - a value _must_ be passed for it in the first position, and if it's `empty()`, then it gets set to a portion of the widget's PHP class name.

I'd suggest that "Default partial class name." isn't added, and "Optional." is removed from this param.

🔢 Applies to `WP_Widget()` below also.

Reply:
Yes, looks like it's been incorrectly changed here: https://github.com/WordPress/wordpress-develop/commit/6c17894e5e3acf8656af34ac2b4002863ebc0d80